### DISCUSSION TITLE
Migrating a drupal 8 site to statamic

### DISCUSSION THREAD

**Discussion by marceloandrader** [statamic/cms#5871] [Q&A]
Hi friends,

I'm considering migrating a site from drupal to statamic, this site is a medium size digital magazine, it has around 23k articles. wondering if this kind of migration has been done in the past, what considerations should I have. Is it ok to manage this size in a flat file or better to use a db, in that case what will be the difference between keeping it in drupal.

Thanks in advance.

**Comment by jasonvarga** [comment]
If you're worried about size, you can store your entries in the database.

https://github.com/statamic/eloquent-driver

Plenty of people are storing lots of data in Statamic.

**Reply by marceloandrader** [reply]
23k files seems like is not too much to me, what is the threshold to use a database?

**Reply by jackmcdade** [reply]
For the front-end with caching it barely matters, but you might find working with 20k+ entries in a single collection a bit sluggish in the control panel.

**Reply by marceloandrader** [reply]
Thanks
